### PR TITLE
Fixed for WARN about insecureNotifications mode

### PR DESCRIPTION
**Pull request by Anjali-NEC**
Fixed for issue #3013

### CODE REVIEW

**Comment by Anjali-NEC** [comment]
Hi @fgalan sir,

Please review my PR.

File: src/app/contextBroker/contextBroker.cpp [hunk 1]
```diff
@@ -606,6 +606,12 @@ static void contextBrokerInit(std::string dbPrefix, bool multitenant)
 
   /* Set HTTP timeout */
   httpRequestInit(httpTimeout);
+  
+  //WARN about insecureNotifications mode
+  if (insecureNotif == true)
+   {
+      LM_W(("WARNING: contextBroker started in insecure mode (-insecureNotif)"));
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp:613]:
```suggestion
      LM_W(("WARNING: contextBroker started in insecure notifications mode (-insecureNotif)"));
```

File: CHANGES_NEXT_RELEASE [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- Fix: WARN about insecureNotifications mode (#3013)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: log WARN about insecureNotifications mode (#3013)
```

**Comment by fgalan** [comment]
> 
> 
> Hi @fgalan sir,
> 
> Please check the PR if it is Ok then please merge it.

I did it and provide a couple of minor comments (which can be directly commited using the buttom in the github interface for that).

Apart from that, it would be great if you could include the Orion trace when it starts in insecure mode to see how the log looks like (e.g. `contectBroker -fg -insecureNotif).

Thanks for your contribution!

**Comment by Anjali-NEC** [comment]
I have updated my PR as per your suggestions. 

> Apart from that, it would be great if you could include the Orion trace when it starts in insecure mode to see how the log looks like (e.g. `contectBroker -fg -insecureNotif).

I am not getting your point can you please explain in detail? please find attach screenshot when I run "contectBroker -fg -insecureNotif"

CB(insecureNotify)

File: src/app/contextBroker/contextBroker.cpp (hunk 1 above)
Comment (Anjali-NEC) [reply src/app/contextBroker/contextBroker.cpp]: Updated

File: CHANGES_NEXT_RELEASE (hunk 2 above)
Comment (Anjali-NEC) [reply CHANGES_NEXT_RELEASE]: Updated

**Comment by fgalan** [comment]
> I am not getting your point can you please explain in detail? please find attach screenshot when I run "contectBroker -fg -insecureNotif"

I refer to the screenshot you have posted. Note that WARN/WARNING appears twice in the message, so I think the message text could be simplified. I'll do a new suggestion comment.

File: src/app/contextBroker/contextBroker.cpp (hunk 1 above)
Comment (fgalan) [reply src/app/contextBroker/contextBroker.cpp]: Fixed in d53756e

File: CHANGES_NEXT_RELEASE (hunk 2 above)
Comment (fgalan) [reply CHANGES_NEXT_RELEASE]: Fixed in 60e8d58

File: src/app/contextBroker/contextBroker.cpp [hunk 3]
```diff
@@ -606,6 +606,12 @@ static void contextBrokerInit(std::string dbPrefix, bool multitenant)
 
   /* Set HTTP timeout */
   httpRequestInit(httpTimeout);
+  
+  //WARN about insecureNotifications mode
+  if (insecureNotif == true)
+   {
+      LM_W(("WARNING: contextBroker started in insecure notifications mode (-insecureNotif)"));
```
Comment (fgalan) [inline src/app/contextBroker/contextBroker.cpp:613]:
```suggestion
      LM_W(("contextBroker started in insecure notifications mode (-insecureNotif)"));
```

Comment (Anjali-NEC) [reply src/app/contextBroker/contextBroker.cpp]: Updated

Comment (fgalan) [reply src/app/contextBroker/contextBroker.cpp]: Fixed in 82d9cc9

**Pull request merged**